docs/elements/hx-busy/index.html

```diff
@@ -11,6 +12,9 @@
       The custom <code>{{page.title}}</code> element renders a spinning
       loading indicator.
     </p>
+    <p>
+      Use as an <code>&lt;inline&gt;</code> element.
```

Comment:
instead of a paragraph, add to top of metadata as "Display: inline"

Reply:
@CITguy, do we want the `:` in `Display: inline`?  The other metatdata description terms do not have them.  

<img width="773" alt="screen shot 2018-08-24 at 5 22 58 pm" src="https://user-images.githubusercontent.com/10120600/44610684-dee76200-a7c2-11e8-8f22-fc054fac6f53.png">
 
I don't mind adding them to the other description terms if that's preferred.